debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MysqlDefaultValueIT.java

```diff
@@ -729,7 +729,7 @@ public void dateAndTimeTest() throws InterruptedException {
     public void timeTypeWithConnectMode() throws Exception {
         config = DATABASE.defaultConfig()
                 .with(MySqlConnectorConfig.SNAPSHOT_MODE, MySqlConnectorConfig.SnapshotMode.INITIAL)
-                .with(MySqlConnectorConfig.TABLE_INCLUDE_LIST, DATABASE.qualifiedTableName("DATE_TIME_TABLE"))
+                .with(MySqlConnectorConfig.TABLE_INCLUDE_LIST, DATABASE.qualifiedTableName("DATE_TIME_TABLE_CONNECT_MODE"))
```

Comment:
@jpechane 
conversion of value of `TIME` type is failed when connect mode if out of range from 00:00:00 to 24:00:00.
in `DATE_TIME_TABLE` table, record consumption is failed because default value is out of range.
previously, this value is replaced to null by `handleUnknownData` in JdbcValueConverters with warning log.
but, in `handleUnknownData` of MysqlValueConverters, it's always throw exception. and the exception is handled by `createValueGenerator` in TableSchemaBuilder according to `event.converting.failure.handling.mode`.
so, I have added `DATE_TIME_TABLE_CONNECT_MODE` table without out of range default value in this test case.

Reply:
I am afraid we cannot do that and we need to find a way how to get back to the previous behaviour.
The reasons is that as stange it might sound MySQL database can emit invalid date. The old behvaiour would mean that Debezium would skip over such data but now the connector will stop processing without a way how to restart it.

Reply:
If it would mean that we'd need to duplicate the code then it is the price we need to pay :-(.

Reply:
@jpechane I have tested more and I missed default value converting failure [here](https://github.com/debezium/debezium/blob/main/debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlDefaultValueConverter.java#L111).
I will consider where we handle default value converting failure. :)

Reply:
If we insert out of range time value to mysql with valid default value, consumption error doesn't occur and just remain warning log because `event.converting.failure.handling.mode` is skip(default).

Reply:
@jpechane it looks complicated :(
in TableSchemaBuilder, `converter.convert(value)` is used in key and value generator and default value parser..
so, I think it's better to handle `event.converting.failure.handling.mode` in `ValueConverter` [here](https://github.com/debezium/debezium/pull/5022/commits/4d80b5210ebe04b95432c12b9f7bc8520c78cca8#diff-0fa5a64cda387e02edbc4db74f552610f349f94f1eed7a05d04e9c184e56b385R80-R102).
and MysqlValueConverters use the method in [converters](https://github.com/debezium/debezium/pull/5022/commits/4d80b5210ebe04b95432c12b9f7bc8520c78cca8#diff-23dc3f586e92bd33654f4014313421f5b93d94044e92b4d1b3337594ed3e485fR244).

then, we can use the mode only in mysql connector and this handling mode can be used in all mysql converters.
I removed `parsingErrorHandler` in MysqlValueConverters because it looks similar with the mode.

it requires many changes about converters.. I'm not sure if this is an appropriate change.

Reply:
@jpechane Add the mode handler when parse default value [here](https://github.com/debezium/debezium/pull/5022/files#diff-3526f88652b3ce58c9afb44542a2faf244ea01c0aa2eeb661f22c941d6e964a4R472)